tests/foreman/ui/test_rhai.py

```diff
@@ -0,0 +1,166 @@
+"""Tests for Red Hat Access Insights
+"""
+
+from robottelo.test import UITestCase
+from fauxfactory import gen_string
+from robottelo.ui.session import Session
+from robottelo.common import conf
+from robottelo.vm import VirtualMachine
+from robottelo.common import manifests
+from robottelo.ui.navigator import Navigator
+from nailgun import entities, entity_mixins
+from unittest import TestCase
+from robottelo.common.constants import (
+    DEFAULT_SUBSCRIPTION_NAME,
+)
+
+
+class RHAITestCase(UITestCase, TestCase):
```

Comment:
UITestCase already inherits from TestCase. Drop the multiple inheritance and just use UITestCase